controllers/sriovnetworkpoolconfig_controller.go

```diff
@@ -159,6 +159,8 @@ func (r *SriovNetworkPoolConfigReconciler) syncOvsHardwareOffloadMachineConfigs(
 	}
 
 	data := render.MakeRenderData()
+	data.Data["OtherOvsConfig"] = "other_config:hw-offload=true"
```

Comment:
here we need to use the info from the CR no?

Reply:
fixed